Cargo.toml

```diff
@@ -16,8 +16,8 @@ num_threads = "0.1.2"
 quickcheck = { version = "1.0.3", default-features = false }
 quickcheck_macros = "1.0.0"
 rand = { version = "0.8.4", default-features = false }
-# <= 1.0.171 due to serde-rs/serde#2538
-serde = { version = ">= 1.0.126, <= 1.0.171", default-features = false }
+# >= 1.0.184 due to serde-rs/serde#2538
+serde = { version = ">= 1.0.184", default-features = false }
```

Comment:
You don't need the `>=`, that's the default.

Reply:
Oh true, yes. Thank you. :heart:

Reply:
(`>=` would actually allow be wrong as it allows updating to v2 etc., the default is `^` which means greater or equal within semver compatibility)